pretext/Functions/intro-Functions.ptx

```diff
@@ -48,22 +47,29 @@
       <li>
         <p>etc.</p>
       </li>
-    </ul>
+     </ul>
   </p>
-  <p>In this chapter you will learn about named functions, functions that can be referred to by name
+  <p>In this chapter you will learn about named functions, i.e. functions that are referred to by a name
     when you want to execute them.</p>
   <subsection xml:id="functions_introduction-to-functions_topics">
     <title>Topics</title>
     <p>
-      <ul>
+      <ul marker="">
```

Comment:
I got rid of the marker="" - if we do it for this one we need to do it for the rest of the chapters.